base/applications/wordpad/lang/ro-RO.rc

```diff
@@ -1,8 +1,16 @@
 /*
- * Copyright 2004 Krzysztof Foltman
- * Copyright 2010 Claudia Cotună
- *                Michael Stefaniuc
- *           2011 Ștefan Fulea
+
+
+
+ * PROJECT:     ReactOS Wordpad
```

Comment:
Since someone recently seems to intend to take over this closed PR: those empty lines are not good.